CoEpi/ui/symptomReport/SymptomReportViewController.swift

```diff
@@ -54,13 +54,13 @@ class SymptomReportViewController: UIViewController, ErrorDisplayer {
 
         // TODO clarify (global) button styles
         viewModel.submitButtonEnabled
-            .map { $0 ? .systemIndigo : .lightGray }
+            .map { $0 ? .clear : .clear }
```

Comment:
To set the button's background permanently we don't need rx. Just set it directly to `.clear` in `viewDidLoad` and remove this observer.

Reply:
this is just a temporary change. @mellowgeek plans to use this for issue #192

Reply:
this was reverted as the request of @mellowgeek. His code has already been merged into develop that addresses this.